packages/cardano-graphql/src/Schema/types/Transaction/Script/NativeScript.ts

```diff
@@ -0,0 +1,21 @@
+import { Field, ObjectType } from 'type-graphql';
+import { NOf } from './nof';
+import { PublicKey } from '../../PublicKey';
+import { Slot } from '../../Slot';
+
+// Review: alternatively could create a union type
```

Comment:
A union which defines the common script fields in a base type?

Reply:
Reusing the fields doesn't make much sense here, as exactly 1 of those fields has to be set. I meant something like:
```typescript
type NativeScriptAny = {type: 'ANY'; scripts: NativeScript[];};
type NativeScriptAll = {type: 'ALL'; scripts: NativeScript[];};
type NativeScript = NativeScriptAny | NativeScriptAll | ...
```